src/modelgauge/main.py

```diff
@@ -253,6 +253,12 @@ def run_test(
     required=False,
     callback=validate_uid,
 )
+@click.option(
+    "--output-path",
```

Comment:
Typically something called a path would be a directory. This might be better called `--output-file` or `--output`.